### DISCUSSION TITLE
Error compiling SimpleStorage contract with foundry zksync

### DISCUSSION THREAD

**Discussion by SpringxDSay** [Cyfrin/foundry-full-course-cu#2255] [Q&A]
While trying to compile my solidity code using the command "forge build --zksync", the error below was returned:
```
[⠃] Using zksolc-1.5.1
[⠒] Compiling (zksync)
[⠢] Compiling 21 files with ZkSolc 0.8.25
Error: 
Broken pipe (os error 32)
```
I made sure I followed the installation process as it is in the video and I am compiling in the right directory. Any idea on what could be wrong?

**Comment by EngrPips** [comment]
Hello @SpringxDSay, Please see this https://github.com/matter-labs/foundry-zksync/issues/453 on the official Foundry repo and maybe it can give hint on how to resolve this issue
